pybinder/core.py

```diff
@@ -2649,6 +2658,17 @@ def generate_method(binder):
     sig = function_signature(binder)
     nargs, ndefaults, args_name, args_type, defaults, is_array_like = sig
 
+    # Pybind return policy
+    if qname in Generator.return_policies:
+        policy = Generator.return_policies[qname]
+        rv_policy = f', py::return_value_policy::{policy}'
+    elif (rtype.rstrip().endswith("&") and
```

Comment:
there is a property in the binder called `is_lvalue`, does that work instead of looking for a string? there is also `is_const_qualified` property, so those two detect what were looking for instead of searching for strings?

Reply:
Should it be like this?

```python
elif binder.rtype.is_lvalue and not binder.rtype.is_const_qualified:
        rv_policy = ', py::return_value_policy::reference'
```

Gave it a try... `is_lvalue` seems to work well but `is_const_qualified` doesn't seem to.